packages/Breadcrumbs/src/components/Link/Link.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Popover from "@paprika/popover";
+import IsDarkContext from "../../context";
+import { MAXIMUM_NUM_OF_CHARACTER } from "../../constants";
+
+import * as sc from "./Link.styles";
+
+const truncate = str =>
+  str.length > MAXIMUM_NUM_OF_CHARACTER ? `${str.substring(0, MAXIMUM_NUM_OF_CHARACTER)}...` : str;
+
+function isString(item) {
+  return typeof item === "string" || item instanceof String;
+}
+
+function Link(props) {
+  const { children, href, as, ...moreProps } = props;
+  const shouldTruncate = isString(children) && children.length > MAXIMUM_NUM_OF_CHARACTER;
+
+  const link = (
+    <IsDarkContext.Consumer>
```

Comment:
`const isDark = React.useContent(IsDarkContext)` ? or why is the reason to use `.Consumer` ?

Reply:
Addressed! resolve the conversation